### PR TITLE
feat: add ai code review PR test

### PR DESCRIPTION
**Pull request by mvn-vinhhuynh-dn**
## Summary by CodeRabbit

- **New Features**
  - Introduced an AI-powered code review process on pull requests.

- **Enhancements**
  - Improved app's responsiveness to system configuration changes and memory constraints.

- **Documentation**
  - Updated workflow documentation to reflect new review automation.

### CODE REVIEW

**Comment by mvn-vinhhuynh-dn** [comment]
run review

**Comment by mvn-vinhhuynh-dn** [comment]
@coderabbitai run review

**Comment by mvn-vinhhuynh-dn** [comment]
@coderabbitai run review ai-pr-reviewer

**Comment by mvn-vinhhuynh-dn** [comment]
@coderabbitai run review ai_code_review

**Comment by mvn-vinhhuynh-dn** [comment]
@coderabbitai run github action workflow name ai_code_review.yml

**Comment by mvn-vinhhuynh-dn** [comment]
run review

**Comment by mvn-vinhhuynh-dn** [comment]
run review

**Review by mvn-toannguyen2-dn — COMMENTED** [review]
run review

File: app/src/main/java/mvn/dn/basecompose/App.kt [hunk 1]
```diff
@@ -8,4 +10,27 @@ import dagger.hilt.android.HiltAndroidApp
...
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        super.onConfigurationChanged(newConfig)
+        Log.d("VVVV", "Test1")
+    }
+
+    override fun onLowMemory() {
+        super.onLowMemory()
+        Log.d("VVVV", "Test2")
+
+    }
```
[the remark this answers is not in the document]
Comment (mvn-toannguyen2-dn) [reply app/src/main/java/mvn/dn/basecompose/App.kt:35]: run review

[the remark this answers is not in the document]
Comment (mvn-vinhhuynh-dn) [reply app/src/main/java/mvn/dn/basecompose/App.kt:35]: @coderabbitai resolve

[the remark this answers is not in the document]
Comment (mvn-vinhhuynh-dn) [reply app/src/main/java/mvn/dn/basecompose/App.kt:35]: run review

[the remark this answers is not in the document]
Comment (mvn-vinhhuynh-dn) [reply app/src/main/java/mvn/dn/basecompose/App.kt:35]: @coderabitai run review

**Pull request closed without merging**